vmf/scripts/mods/vmf/modules/ui/options/vmf_options_view.lua

```diff
@@ -855,31 +904,57 @@ local function create_header_widget(widget_definition, scenegraph_id)
         --]]
         ---[[
         {
-          pass_type = "texture",
+          pass_type = "triangle",
```

Comment:
One of the objectives of VMF is to be cross-compatible between VT1 and VT2. This and 3 next passes could not be processed by VT1, because VT1 lacks `triangle` pass.

Reply:
The triangle pass isn't maskable either I think. Best option I see is bring in a small arrow texture from the internet